
# remove a part from the world

string name

---

bool success
string message
